tests/unittests.c

```diff
@@ -160,18 +167,22 @@ int unit_tests(void *_unused) {
     cpu_freq_expect("Prototyp Amazing Foo One @ 1GHz", 1000000000);
     cpu_freq_expect("Prototyp Amazing Foo Two @ 1.00GHz", 1000000000);
 
-    task_t *task1, *task2, *task_user1, *task_user2;
+    task_t *task1, *task2, *task_user1, *task_user2, *task_user3;
 
     task1 = new_kernel_task("test1", test_kernel_task_func, _ptr(98));
     task2 = new_kernel_task("test2", test_kernel_task_func, _ptr(-99));
     task_user1 = new_user_task("test1 user", test_user_task_func1, NULL);
     task_user2 = new_user_task("test2 user", test_user_task_func2, NULL);
+    task_user3 = new_user_task("test3 user", test_user_task_func3, NULL);
+
+    vmap_user_4k(HIGH_USER_PTR, get_free_frame()->mfn, L1_PROT_USER);
```

Comment:
nit: maybe add additional kernel mapping first for a nearby 4k page to populate with PT with superuser entries.